### DISCUSSION TITLE
help help...how to turn off the default generated menu bar

### DISCUSSION THREAD

**Discussion by xiaoqiangclub** [beeware/toga#1477] [Q&A] [answered]
hello,
I am a newbie and I would like to ask how to turn off the default generated menu bar (file and Help).
Or how can I change the default 2 menus .
thank you！
![image](https://user-images.githubusercontent.com/61998040/166150441-67267efd-5e59-40c7-9b95-aada7142de6a.png)

**Comment by freakboy3742** [comment, marked as answer]
In short - at the moment, you don't. Toga is currently focussed on the use case of building a desktop-native app; App-level menus that expose (at the very least) quit and help functions are a key part of what an app needs on every platform. You can add new items to the menus, and you can add entirely new top level menus - but you can't remove the base "Quit" and "About" menu items. That means that Toga isn't a good match for use cases like games that don't want this sort of window decoration.

One idea that has been suggested (#733) is the idea of a "SimpleApp" - an app that doesn't expose menus, toolbars, or anything else. That doesn't exist at present, but I'd be in favour of adding it if someone was motivated to work up a PR that implemented it.

**Reply by xiaoqiangclub** [reply]
I see. Thank you.

**Comment by t-arn** [comment]
In my apps, I use 
`self.app.commands=toga.CommandSet(self.app.factory)` to replace the standard menu with my localized one. If you don't repopulate the CommandSet, it should be invisible (I think...)

**Reply by xiaoqiangclub** [reply]
Thank you. That's what I want.
